### PR TITLE
[REVIEW] Pin `dask` and `distributed` for release

### PR DESCRIPTION
**Pull request by galipremsagar**
## Description
This PR pins `dask` and `distributed` to `2022.9.2` for `22.10` release.

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).

### CODE REVIEW

**Review by raydouglass — APPROVED** [review]

**Review by wence- — APPROVED** [review]
Think we are holding for `2022.9.2`? Otherwise 👍

File: ci/benchmark/build.sh [hunk 1]
```diff
@@ -37,10 +37,10 @@ export GBENCH_BENCHMARKS_DIR="$WORKSPACE/cpp/build/gbenchmarks/"
 export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 
 # Dask & Distributed option to install main(nightly) or `conda-forge` packages.
-export INSTALL_DASK_MAIN=1
+export INSTALL_DASK_MAIN=0
 
 # Dask version to install when `INSTALL_DASK_MAIN=0`
-export DASK_STABLE_VERSION="2022.7.1"
+export DASK_STABLE_VERSION="2022.9.1"
```
Comment (galipremsagar) [inline ci/benchmark/build.sh]:
```suggestion
export DASK_STABLE_VERSION="2022.9.2"
```

**Comment by galipremsagar** [comment]
rerun tests

File: python/cudf/cudf/tests/test_orc.py [hunk 2]
```diff
@@ -736,6 +736,7 @@ def test_orc_write_statistics(tmpdir, datadir, nrows, stats_freq):
 @pytest.mark.parametrize("stats_freq", ["STRIPE", "ROWGROUP"])
 @pytest.mark.parametrize("nrows", [2, 100, 6000000])
 def test_orc_chunked_write_statistics(tmpdir, datadir, nrows, stats_freq):
+    np.randon.seed(0)
```
Comment (galipremsagar) [inline python/cudf/cudf/tests/test_orc.py:739]:
```suggestion
    np.random.seed(0)
```

**Review by raydouglass — APPROVED** [review]

**Review by shwina — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

**Comment by galipremsagar** [comment]
@gpucibot merge

**Pull request merged**